uiServices/src/testCaseServices_test.go

```diff
@@ -414,4 +445,84 @@ func TestGetTestCaseFileNotFound(t *testing.T) {
 	}
 
 	assert.Equal(t,http.StatusNotFound, w.Code, "Should not return data")
+}
+
+
+
+func TestSuccessfulCaseDelete(t *testing.T) {
+	r := chi.NewRouter()
+	r.Mount("/", GetIndexPage())
+
+	//create file to be deleted
```

Comment:
Try to avoid commenting on your go code. Instead try to write the code in such a way that the reader of your code understands it without need for comments. 
A good read on the topic would be "Clean Code"  by Robert C. Martin. It is available on Safari books.